tests/test_build_wf.py

```diff
@@ -208,7 +208,7 @@ async def test_execute_step(context: StreamFlowContext):
             port_target=None,
             port_type="string",
             cwl_element={},
-            context={"hints": {}, "requirements": {}},
+            context={"hints": {}, "requirements": {}, "version": "v1.2"},
```

Comment:
```suggestion
            context={"hints": {}, "requirements": {}, "version": CWL_VERSION},
```

from tests.utils.workflow import CWL_VERSION